paperqa/settings.py

```diff
@@ -482,35 +494,24 @@ def from_name(
 
         raise FileNotFoundError(f"No configuration file found for {config_name}")
 
-    def _default_litellm_router_settings(self, llm: str) -> dict:
-        """Settings matching "model_list" schema here: https://docs.litellm.ai/docs/routing."""
-        return {
-            "model_list": [
-                {
-                    "model_name": llm,
-                    "litellm_params": {"model": llm, "temperature": self.temperature},
-                }
-            ]
-        }
-
     def get_llm(self) -> LiteLLMModel:
         return LiteLLMModel(
             name=self.llm,
-            config=self.llm_config or self._default_litellm_router_settings(self.llm),
+            config=self.llm_config or make_default_litellm_router_settings(self.llm),
```

Comment:
can you put self.temperature as an input into these now that it's a free function?

Reply:
Alright, just pushed this fix, thanks again